platform/security/policy/security-policy-context/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -29,10 +29,10 @@
             type="String"
             default="/=karaf"/>
 
-        <AD description="List of authentication types required for each context. List of default valid authentication types are: IDP, SAML, BASIC, PKI, GUEST. Example: /context=AUTH1|AUTH2|AUTH3"
+        <AD description="List of authentication types required for each context. List of default valid authentication types are: SAML, BASIC, PKI, GUEST. Example: /context=AUTH1|AUTH2|AUTH3"
             name="Authentication Types" id="authenticationTypes" required="true" cardinality="100"
             type="String"
-            default="/=IDP|GUEST,/solr=SAML|PKI|BASIC"/>
+            default="/=SAML|GUEST,/admin=SAML|basic,/system=basic,/solr=SAML|PKI|basic,/sources=SAML|basic,/security-config=SAML|basic"/>
```

Comment:
❓ Change to the metatype not reflected in the blueprint as well (https://github.com/AzGoalie/ddf/blob/bad47c3254d6f08a43cab2a2dcfdc7992ccdf81d/platform/security/policy/security-policy-context/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L26). Should they be synched?